app/assets/javascripts/components/locales/fr.jsx

```diff
@@ -64,23 +75,31 @@ const fr = {
   "navigation_bar.favourites": "Favoris",
   "navigation_bar.info": "Plus d'informations",
   "navigation_bar.logout": "Déconnexion",
+  "navigation_bar.follow_requests": "Demandes de suivi",
   "reply_indicator.cancel": "Annuler",
-  "search.placeholder": "Chercher",
+  "search.placeholder": "Rechercher",
   "search.account": "Compte",
   "search.hashtag": "Mot-clé",
   "search_results.total": "{count} {count, plural, one {résultat} other {résultats}}",
+  "search.status_by": "Statuts de {name}",
   "upload_button.label": "Joindre un média",
   "upload_form.undo": "Annuler",
+  "upload_progress.label": "Envoi en cours…",
+  "upload_area.title": "Glissez et déposez pour envoyer",
   "notification.follow": "{name} vous suit.",
   "notification.favourite": "{name} a ajouté à ses favoris :",
   "notification.reblog": "{name} a partagé votre statut :",
   "notification.mention": "{name} vous a mentionné⋅e :",
   "notifications.column_settings.alert": "Notifications locales",
   "notifications.column_settings.show": "Afficher dans la colonne",
+  "notifications.column_settings.sound": "Émettre un son",
   "notifications.column_settings.follow": "Nouveaux abonnés :",
   "notifications.column_settings.favourite": "Favoris :",
   "notifications.column_settings.mention": "Mentions :",
   "notifications.column_settings.reblog": "Partages :",
+  "notifications.clear": "Nettoyer",
+  "notifications.clear_confirmation": "Êtes-vous certain de vouloir nettoyer toutes vos notifications ?",
```

Comment:
The usual workaround is "Voulez-vous vraiment nettoyer toutes vos notifications ?" because it is gender-neutral :)